src/index.tsx

```diff
@@ -2,12 +2,23 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './css/bootstrap.scss';
 import App from './App'; // Keep temporarily or linter will complain
-
-import { RiderSignup } from "./Components/Rides/RiderSignup/RiderSignup";
+import { render } from 'react-dom';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { Rides } from './Components/Rides/Rides';
+import { RiderSignup } from './Components/Rides/RiderSignup/RiderSignup';
+import { DriverSignup } from './Components/Rides/DriverSignup/DriverSignup';
+import { Landing } from './Components/Landing/Landing';
 
 ReactDOM.render(
   <React.StrictMode>
-    <RiderSignup />
+    <BrowserRouter>
```

Comment:
Don't worry about creating all the other routes - we'll handle it piece-by-piece (i.e. when we create the pages, we'll add the routes in).